packages/visibility_detector/lib/src/render_visibility_detector.dart

```diff
@@ -140,7 +140,10 @@ mixin RenderVisibilityDetectorBase on RenderObject {
     }
     bool isFirstUpdate = _updates.isEmpty;
     _updates[key] = () {
-      _fireCallback(layer, bounds);
+      if (bounds == null) {
```

Comment:
Maybe leave a comment here explaining to future readers what null bounds mean and why it is ok to not fire the callback in that case?

Reply:
Done